test/linalg/symmetric.jl

```diff
@@ -296,10 +297,14 @@ end
         @testset "linalg binary ops" begin
             @testset "mat * vec" begin
                 @test Symmetric(asym)*x+y ≈ asym*x+y
+                xadj = x.'
+                @test xadj * Symmetric(asym).' ≈ xadj * asym
                 @test x' * Symmetric(asym) ≈ x' * asym
 
                 @test Hermitian(aherm)*x+y ≈ aherm*x+y
+                xadj = x'
                 @test x' * Hermitian(aherm) ≈ x' * aherm
+                @test xadj * Hermitian(aherm)' ≈ xadj * aherm
```

Comment:
This test seems to check only that `adjoint(Hermitian(aherm))` is approximately `Hermitian(aherm)`?

Reply:
I'm after testing [these fallbacks](https://codecov.io/gh/JuliaLang/julia/src/master/base/linalg/symmetric.jl#L318) - will this not do it?

Reply:
If this test exercises the `*(A::Hermitian, x::Bool)` fallback, I miss how? Also, if you substitute `x'` for `xadj` in this test, the result is `x' * Hermitian(aherm)' ≈ x' * aherm`, which differs from the immediately preceding test only in the `'` on `Hermitian(aherm)'`? Am I missing something? :)

Reply:
> If this test exercises the *(A::Hermitian, x::Bool) fallback, I miss how?

That's what I thought first too, but the link points to L318, but my browser does not end up there...

Reply:
That makes much more sense! Thank you! :)

Reply:
Alright, I see the intent now :). Perhaps a comment or two describing the intent would be worthwhile? Perhaps something along the lines of `@test (xadj = x'; xadj * Hermitian(aherm)' ≈ xadj * aherm) # exercise A_mul_Bc(RowVector, Hermitian)`?

Reply:
can do :) sorry about the weird linking issues

Reply:
Not your fault! :)